src/OpenLoco/StationManager.cpp

```diff
@@ -162,4 +354,16 @@ namespace OpenLoco::StationManager
             }
         }
     }
+
+    void registerHooks()
+    {
+        // Can be removed once the createStation function has been implemented (used by place.*Station game commands)
+        registerHook(
+            0x048F988,
+            [](registers& regs) FORCE_ALIGN_ARG_POINTER -> uint8_t {
+                StationId_t stationId = static_cast<StationId_t>((0x005E6EDC - regs.esi) / sizeof(Station));
```

Comment:
esi is a station pointer think you could just use the station.id() method instead.
```cpp
auto stationId = (reinterpret_cast<Station*>(regs.esi))->id();
```